zombies.js

```diff
@@ -111,6 +150,21 @@
  * @return {boolean} true/false     Whether player was able to store item in pack.
  */
 
+Player.prototype.takeItem = function(item){
+  if (this.getPack().includes(item)){
```

Comment:
I don't think this is necessary since it's just adding the same item back into the pack? If it's already in the pack, then it shouldn't be added back into the pack. It's already in the pack. Remember that `.includes` does a `===` match for objects. Which means if this is true, then the item is literally already in the array (or pack in this case).